.eslintrc.js

```diff
@@ -112,8 +112,30 @@ module.exports = {
         '@typescript-eslint/naming-convention': 'off',
         '@typescript-eslint/no-unused-expressions': 'off',
         'prefer-arrow/prefer-arrow-functions': 'off',
+        'import/no-duplicates': 'error',
+        'import/no-unused-modules': 'error',
         'import/no-unassigned-import': 'error',
-        'import/order': 'error',
+        'import/order': [
+          'error',
+          {
+            alphabetize: { order: 'asc', caseInsensitive: false },
+            'newlines-between': 'always',
+            groups: ['builtin', 'external', 'internal', [ 'parent', 'sibling', 'index']],
+            pathGroups: [
+              {
+                pattern: '{@angular/**,rxjs,rxjs/operators}',
+                group: 'builtin',
```

Comment:
I think `group` there should be `external`, because `builtin` represent node modules

And `pathGroupsExcludedImportTypes` defines import types(default: "builtin", "external") that are not handled by configured pathGroups, so it's confused to use `["ng-zorro-antd"]`, it just work as `[]`. Maybe use `["builtin"]` or `[]` more appropriate

Reply:
I think `external` rather represents node modules and `builtin` just refers to things like `fs` and `path`. However, you are correct about `"pathGroupsExcludedImportTypes"`, `[]` would be sufficient and more appropriate in this case.

Reply:
Yea, you are right and the **node modules** I mean node system special module, just like `fs` that you mention.
So why treat `@angular/**,rxjs,rxjs/operators` as `builtin`, it will cause a mix of `builtin` and `external`.
If just want `@angular/**,rxjs,rxjs/operators` to be top of import declaration, `position: 'before'` may be enough.

Reply:
Got your point, thx!